frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -201,7 +203,7 @@ export const Exchange = ({ code, accounts }: TProps) => {
                       },
                       ...regions]
                       }
-                      value={selectedRegion || ''}
+                      value={selectedRegion}
```

Comment:
Not actually doing anything here :) just a cleanup. Since selectedRegion is already always a string, that `|| ''` is useless.